drakrun/drakrun/draksetup.py

```diff
@@ -390,44 +393,38 @@ def send_usage_report(report):
         logging.exception("Failed to send usage report. This is not a serious problem.")
 
 
-def create_rekall_profiles(install_info: InstallInfo):
-    storage_backend = get_storage_backend(install_info)
-    with storage_backend.vm0_root_as_block() as block_device, \
-            tempfile.TemporaryDirectory() as mount_path:
-        mnt_path_quoted = shlex.quote(mount_path)
-        blk_quoted = shlex.quote(block_device)
+def create_rekall_profiles(install_info: InstallInfo, runtime_info: RuntimeInfo, kernel_profile: str):
```

Comment:
One small nitpick: install_info now becomes redundant.
I would pass `Injector` directly into this function

Reply:
Sure, will change it